Changed fonts, results_rendering

lennessyy opened the pull request:
It's only some small changes, and hopefully they work (I tested them by copying the JS to the developer console since I don't know how to start a server using Go yet, and they worked on my end so they should be okay)

ishunyu commented on static/fonts/style.css:
```diff
@@ -0,0 +1,120 @@
+/* #### Generated By: http://www.cufonfonts.com #### */
```
Hey @lennessyy, not sure if I understand why there are so many different BAHNSCHRIFT.woff files? They seems to be the same.

lennessyy replied:
I downloaded them online and assumed they they were different weights or something without looking into them deeper. Just deleted the duplicates.

ishunyu commented on static/index.html:
```diff
@@ -1,135 +1,24 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Appreciate the refactor.

ishunyu commented:
Can you also remove the duplicate .woff files in `static/fonts`?

lennessyy commented:
Just deleted! I actually thought I deleted them before but clearly I still don't know how to use github lol (this was actually my first actual pull request)

ishunyu commented:
No worries, looks good!

ishunyu commented:
Feel free to merge now.
